### PR TITLE
Revised teams exercises and added solutions

### PR DESCRIPTION
**Pull request by lwjohnst86**
Just so others know what's being added/modified. I didn't really add any new exercises here, since I could think of any more to add for this chapter.

### CODE REVIEW

File: chapters/solutions.Rmd [hunk 1]
```diff
@@ -743,21 +745,86 @@ We also have a code of conduct (see [`CONDUCT.md`](CONDUCT.md)).
...
+is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By
+contributing to this project, you agree to abide by its terms.
+```
 
 ### Exercise \@ref(teams-ex-balancing)
 
-FIXME
+Some solutions could be:
+
+- Give the team member their own office space so they don't distract others
```
Comment (gvwilson) [inline chapters/solutions.Rmd:814]: '.' at the end of bullet points, since these are sentences.

File: chapters/solutions.Rmd [hunk 2]
```diff
@@ -743,21 +745,86 @@ We also have a code of conduct (see [`CONDUCT.md`](CONDUCT.md)).
...
+- Re-arrange the work spaces so that there is a "quiet office" and a regular
+office space and have the team member with the attention disorder work
+in the regular office
 
 ### Exercise \@ref(teams-ex-contributions)
 
-FIXME
+Possible solutions:
+
+- Change the rule so that anyone who contributes to the project, in any way,
```
Comment (gvwilson) [inline chapters/solutions.Rmd:824]: '.' at the ends of bullet points, since these are sentences.

File: chapters/teams.Rmd [hunk 3]
```diff
@@ -1349,7 +1362,7 @@ if someone's name appears in the Git history for a project,
 they are listed as a co-author on papers for that project.
 A new member of your team has complained that this is unfair:
 people who haven't contributed for over two years are still being included as authors,
-while they aren't even though they have spent many hours doing code reviews of others' work.
+while they aren't included because their contributions of doing many hours of code reviews don't show up in the Git history.
```
Comment (gvwilson) [inline chapters/teams.Rmd:1365]: "because the many hours they have spent doing code reviews don't show up in the Git history"

**Comment by lwjohnst86** [comment]
Done. Merging.

**Pull request merged**